src/jquery.addressfield.js

```diff
@@ -65,6 +65,9 @@
         url: configs.json,
         async: configs.async,
         success: function (data) {
+          if($(configs.fields.country)[0].children.length === 0){
```

Comment:
I feel like this is not optimal, what is the best practice way to check for option fields here?